internal/interpreter/dynamic_component.rs

```diff
@@ -930,6 +930,7 @@ pub(crate) fn generate_component<'id>(
                 "TextVerticalAlignment" => {
                     property_info::<i_slint_core::items::TextVerticalAlignment>()
                 }
+                "InputType" => { property_info::<i_slint_core::items::InputType>() }
```

Comment:
I think in addition you may need to add

`declare_value_enum_conversion!(i_slint_core::items::InputType, InputType);`

to `interpreter/api.rs` to fix the errors seen in the CI.